tasks/pythagoras-tree/pythagoras_tree.svg

```diff
@@ -0,0 +1 @@
olygon><polygon points=' -562 -950 -556 -956 -550 -956 -550 -950 -556 -956 -550 -950 -556 -944'></polygon><polygon points=' -556 -944 -550 -938 -550 -931 -556 -931 -550 -938 -556 -931 -562 -938'></polygon><polygon points=' -575 -925 -569 -919 -569 -912 -575 -912 -569 -919 -575 -912 -581 -919'></polygon><polygon points=' -581 -919 -588 -912 -594 -912 -594 -919 -588 -912 -594 -919 -588 -925'></polygon><polygon points=' -600 -912 -594 -919 -588 -919 -588 -912 -594 -919 -588 -912 -594 -906'></polygon><polygon points=' -594 -906 -588 -900 -588 -894 -594 -894 -588 -900 -594 -894 -600 -900'></polygon><polygon points=' -612 -888 -606 -881 -606 -875 -612 -875 -606 -881 -612 -875 -619 -881'></polygon><polygon points=' -619 -881 -625 -875 -631 -875 -631 -881 -625 -875 -631 -881 -625 -888'></polygon><polygon points=' -650 -888 -644 -881 -644 -875 -650 -875 -644 -881 -650 -875 -656 -881'></polygon><polygon points=' -656 -881 -662 -875 -669 -875 -669 -881 -662 -875 -669 -881 -662 -888'></polygon><polygon points=' -675 -900 -681 -894 -688 -894 -688 -900 -681 -894 -688 -900 -681 -906'></polygon><polygon points=' -681 -906 -688 -912 -688 -919 -681 -919 -688 -912 -681 -919 -675 -912'></polygon><polygon points=' -675 -912 -669 -919 -662 -919 -662 -912 -669 -919 -662 -912 -669 -906'></polygon><polygon points=' -669 -906 -662 -900 -662 -894 -669 -894 -662 -900 -669 -894 -675 -900'></polygon><polygon points=' -688 -888 -681 -881 -681 -875 -688 -875 -681 -881 -688 -875 -694 -881'></polygon><polygon points=' -694 -881 -700 -875 -706 -875 -706 -881 -700 -875 -706 -881 -700 -888'></polygon><polygon points=' -725 -888 -719 -881 -719 -875 -725 -875 -719 -881 -725 -875 -731 -881'></polygon><polygon points=' -731 -881 -738 -875 -744 -875 -744 -881 -738 -875 -744 -881 -738 -888'></polygon><polygon points=' -750 -900 -756 -894 -762 -894 -762 -900 -756 -894 -762 -900 -756 -906'></polygon><polygon points=' -756 -906 -762 -912 -762 -919 -756 -919 -762 -912 -756 -919 -750 -912'></polygon><polygon points=' -762 -925 -756 -919 -756 -912 -762 -912 -756 -919 -762 -912 -769 -919'></polygon><polygon points=' -769 -919 -775 -912 -781 -912 -781 -919 -775 -912 -781 -919 -775 -925'></polygon><polygon points=' -788 -938 -794 -931 -800 -931 -800 -938 -794 -931 -800 -938 -794 -944'></polygon><polygon points=' -794 -944 -800 -950 -800 -956 -794 -956 -800 -950 -794 -956 -788 -950'></polygon><polygon points=' -788 -975 -794 -969 -800 -969 -800 -975 -794 -969 -800 -975 -794 -981'></polygon><polygon points=' -794 -981 -800 -988 -800 -994 -794 -994 -800 -988 -794 -994 -788 -988'></polygon><polygon points=' -775 -1000 -781 -1006 -781 -1012 -775 -1012 -781 -1006 -775 -1012 -769 -1006'></polygon><polygon points=' -769 -1006 -762 -1012 -756 -1012 -756 -1006 -762 -1012 -756 -1006 -762 -1000'></polygon><polygon points=' -750 -988 -744 -994 -738 -994 -738 -988 -744 -994 -738 -988 -744 -981'></polygon><polygon points=' -744 -981 -738 -975 -738 -969 -744 -969 -738 -975 -744 -969 -750 -975'></polygon><polygon 
```

Comment:
Let's avoid committing the SVG file itself, I would consider it like a build artifact.

Reply:
Moved pythagoras_tree.svg to personal gist.
I need a link to the result for http://rosettacode.org/wiki/Pythagoras_tree#Rust.
BTW, please tell me how to upload image files directly to rosettacode.org: every time I try it returns 'Internal error: fatal exception of type "UploadStashFileException" '